### DISCUSSION TITLE
[3 luglio 2024] Calcolabilità

### DISCUSSION THREAD

**Discussion by Exyss** [sapienzastudentsnetwork/automi-calcolabilita-complessita#32] [Calcolabilità]
![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/65043207/408e6032-6985-4a36-8ac2-8832689626a6)

**Comment by Exyss** [comment]
Per il primo punto, ripropongo la soluzione presente sui miei appunti:

![image](https://github.com/user-attachments/assets/d505e991-7012-4568-b561-742abda75243)

----

La dimostrazione richiesta nel secondo punto è presente sul libro di testo

**Comment by daromacarlo** [comment]
<img width="1559" height="388" alt="immagine" src="https://github.com/user-attachments/assets/82b976ea-593c-45b5-bddb-dff85f09f973" />
